static/js/gtag.js

```diff
@@ -18,9 +18,17 @@ window.addEventListener('DOMContentLoaded', function () {
     // Events on links should be sent via beacon, see:
     // https://support.google.com/analytics/answer/7478520?hl=en
     if (elem.hasAttribute('href')) {
-      data.event_callback = function() { document.location = elem.href; };
+      var updated = false;
+      var callbackFn = function() {
+        if (updated) return;
+        updated = true;
+        document.location = elem.href;
+      };
+      data.event_callback = callbackFn;
       data.transport_type = 'beacon';
-      e.preventDefault();
```

Comment:
If we don't prevent the default behaviour, it will try to use gtag to log the event, but very likely the browser will interrupt that by immediately loading the new page. Some events may be logged, but most will be lost.

Reply:
But you remove the line that prevents default behavior?